### PR TITLE
i18n for public member profile date format

### PR DESCRIPTION
**Pull request by deckerweb**
- add i18n for date format in public member profiles
- this way displayed date format can also be translated/ customized

### CODE REVIEW

File: gp-templates/profile-public.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 				}
 			?></dd>
 			<dt><?php _e( 'Member Since', 'glotpress' ); ?></dt>
-			<dd><?php echo date( 'M j, Y', strtotime( $user->user_registered ) ); ?></dd>
+			<dd><?php /* translators: public profile date format */ echo date_i18n( __( 'M j, Y', 'glotpress' ), strtotime( $user->user_registered ) ); ?></dd>
```
Comment (toolstack) [inline gp-templates/profile-public.php]: To resolve the failed build, add `// WPCS: XSS ok.` after the `date_i18n()` call.

Also, maybe use `_x()` to give the context.

Comment (swissspidy) [reply gp-templates/profile-public.php]: A comment makes more sense than `_x()` IMHO, though it should be on a new line for better readability.

Comment (toolstack) [reply gp-templates/profile-public.php]: One does not exclude the other... also see #99.

Comment (SergeyBiryukov) [reply gp-templates/profile-public.php]: > A comment makes more sense than `_x()`

Agreed. Context would be redundant, unless the same string is used elsewhere in another context.

Comment (deckerweb) [reply gp-templates/profile-public.php]: @toolstack Thanks for the info, I made formatting changes and added the WPCS comment. I hope I did it correct this time (the WPCS comment is new for me in this context). And yes, I agree with the readability :)

As for the comment vs. usage of _x() --- as you all said, comment would be enough for now, only if we have more instances of this date appearance we should consider _x(). I currently did not run in other appearances of dates on the frontend so far... :)

File: gp-templates/profile-public.php [hunk 2]
```diff
@@ -24,7 +24,10 @@
 				}
 			?></dd>
 			<dt><?php _e( 'Member Since', 'glotpress' ); ?></dt>
-			<dd><?php echo date( 'M j, Y', strtotime( $user->user_registered ) ); ?></dd>
+			<dd><?php
+				/* translators: public profile date format */
```
Comment (ocean90) [inline gp-templates/profile-public.php]: Please append `, see https://secure.php.net/date` to the comment. Thanks!

Comment (deckerweb) [reply gp-templates/profile-public.php]: Thanks, just added and commited.
Sorry for the delay, though

**Pull request merged**